include/envoy/upstream/upstream.h

```diff
@@ -283,6 +283,7 @@ class PrioritySet {
   COUNTER  (upstream_cx_http2_total)                                                               \
   COUNTER  (upstream_cx_connect_fail)                                                              \
   COUNTER  (upstream_cx_connect_timeout)                                                           \
+  COUNTER  (upstream_cx_idle_timeout)                                                              \
```

Comment:
Sorry forgot this before. Can you do a follow up doc PR to add this stat to here: https://www.envoyproxy.io/docs/envoy/latest/configuration/cluster_manager/cluster_stats ?

Reply:
will add